src/scheduler.py

```diff
@@ -40,6 +41,10 @@
     cron_command = f"cd /app && PIPENV_PIPFILE={pipfile} {pipenv} run python3 -u {script_public_services} >> {logfile} 2>&1"
     cron.new(command=cron_command).every(1).hours()
 
+    # Set up job for harvesting of information models
+    cron_command = f"cd /app && PIPENV_PIPFILE={pipfile} {pipenv} run python3 -u {script_informationmodels} >> {logfile} 2>&1"
+    cron.new(command=cron_command).every(6).hours()
```

Comment:
Er det ein spesiell grunn til at å høste så sjelden som kvar 6.time? Er for så vidt ok, men ein kan jo vurdere om kvar time er like greit.

Reply:
Tenkte det var greit siden fdk-dataservice-harvester kommer til å sende samme melding når den har endringer, med hver 6. som en backup. Men kan jo ta hver time om det er bedre.